---
layout: page
title: Teaching
group: navigation
permalink: "teaching.html"
---
{% include JB/setup %}

**Biostatistical computing with R**, BIOS524, Instructor/Course director. This course is an introduction to R programming. It covers R fundamentals and statistical functions, R packages, data management/manipulation and plotting using Tidyverse, reproducible research with GitHub, interactive apps with Shiny. [Course website](https://bios524-biostatistical-computing-r.netlify.app/)

**HiCcompareWorkshop**, Detection of Differentially Interacting Chromatin Regions From Multiple Hi-C Datasets. This workshop introduces methods for the comparative (aka differential) analysis of the three-dimensional (3D) structure of the genome using data generated by high-throughput chromatin conformation capture (Hi-C) technologies. [Workshop website](https://mdozmorov.github.io/HiCcompareWorkshop/), [GitHub](https://github.com/mdozmorov/HiCcompareWorkshop), [Docker](https://hub.docker.com/repository/docker/mdozmorov/hiccompareworkshop), [Slides](https://mdozmorov.github.io/Talk_HiCcompare/index.html)

**Deep learning with R**, BIOS691, Instructor/Course director. This course is an introduction to deep learning theory and practice. It will cover the basics of neural network architectures, main statistical concepts behind training neural networks, and implementation aspects. The main focus will be on programming deep neural networks using TensorFlow and its Keras front-end in R, although the knowledge will also be useful for Python practitioners. [Course website](https://bios691-deep-learning-r.netlify.app/)

**Reproducible research tools**,	BIOS691, Summer 2016-18,	Instructor/Course director. This 1-credit hour course introduces the fundamental concepts in computational reproducible research. Through lectures and hands-on exercises students learn best practices of statistical data analysis and programming. [Course website](https://mdozmorov.github.io/BIOS691.2018/)

**Statistical Methods for High-throughput Genomic Data I, II**,	BIOS567/BIOS668, Fall/Spring 2016-18,	Instructor/Course director. This two-parts 3-credit hour course explains basic statistical genomics concepts, from microarray- and sequencing technologies through statistical methods for high-dimensional data analysis. Topics include R/Bioconductor programming, QC, normalization, differential expression analysis, dimensionality reduction, clustering, genomic alignment, variant calling, microRNA-seq, bulk and single-cell RNA-seq, ChIP-seq, methylation, microbial genomics, chromatin conformation capture data analysis. [Course website I](https://mdozmorov.github.io/BIOS567.2018/), [Course website II](https://mdozmorov.github.io/BIOS668.2018/)

**Molecular Biology Genetics and Epigenetics in Psychiatry**,	PSYCH691, Spring 2017,	Guest lecturer. RNA-seq technology, data analysis and interpretation

**Advanced Human Genetics**, HGEN502, Fall/Spring 2017-19,	Guest lecturer. RNA-seq technology, data analysis and interpretation

**Bioinformatics 101**,	Seminar series, Fall 2015-18,	Instructor. This series of 10 lectures include introductory topics on bioinformatics. Topics taught include statistics of enrichment analysis, RNA- and ChIP-seq, methylation technology and analysis, reproducible research.

**Microarray data analysis using R/Bioconductor**,	Workshop, Winter 2013,	Instructor. This 4-day workshop provided in-depth hands-on learning of practical aspects of microarray data analysis and interpretation using R/Bioconductor. Place: Trivandrum, India. [Course material](https://github.com/mdozmorov/ci-workshop)
